Makefile

```diff
@@ -58,7 +58,7 @@ all:	$(TARGETS)
 
 clean:
 	cd libopencm3 && make --no-print-directory clean && cd ..
-	rm -f *.elf *.bin
+	rm -r build_*
```

Comment:
I was thinking that we can cut down on some of the build noise

You could do something like:
[0007-Cut-down-on-noise.patch.txt](https://github.com/PX4/Bootloader/files/1122616/0007-Cut-down-on-noise.patch.txt)